reference/api/error_codes.md

```diff
@@ -0,0 +1,38 @@
+# Error Codes returned by APIs
+
+For http calls made to Dapr runtime, when an error is encountered, an error json is returned in http response body. The json contains an error code and an descriptive error message, e.g.
+```
+{
+    "errorCode": "ERR_STATE_GET",
+    "message": "Requested state key does not exist in state store."
+}
+```
+
+Following table lists the error codes returned by Dapr runtime:
+
+Error Code | Description
+--------- | -----------
+ERR_ACTOR_INSTANCE_MISSING | Error getting an actor instance. This means that actor is now hosted in some other service replica.
```

Comment:
Doesn't have to be here, but we should publish which ones are retryable vs not.  For example the this first one is.

Reply:
@LMWF  thats a good point, we can add them to this list in future.